docs/docsite/rst/user_guide/intro_getting_started.rst

```diff
@@ -109,6 +109,14 @@ You can read more about privilege escalation in :ref:`become`.
 
 Congratulations! You have contacted your nodes using Ansible. You used a basic inventory file and an ad-hoc command to direct Ansible to connect to specific remote nodes, copy a module file there and execute it, and return output. You have a fully working infrastructure.
 
+Resources
+=================================
+* [Product Demos](https://github.com/ansible/product-demos)
+* [Katakoda](https://katacoda.com/rhel-labs)
+* [Workshops](https://github.com/ansible/workshops)
+* [Ansible Examples](https://github.com/ansible/ansible-examples)
+* [Ansible Baseline](https://github.com/ansible/ansible-baseline)
```

Comment:
Could you please change to rst format?

```rst
- `Product Demos <https://github.com/ansible/product-demos>`_
- `Katakoda <https://katacoda.com/rhel-labs>`_
- `Workshops <https://github.com/ansible/workshops>`_
- `Ansible Examples <https://github.com/ansible/ansible-examples>`_
- `Ansible Baseline <https://github.com/ansible/ansible-baseline>`_
```
-  `Product Demos <https://github.com/ansible/product-demos>`_
- `Katakoda <https://katacoda.com/rhel-labs>`_
- `Workshops <https://github.com/ansible/workshops>`_
- `Ansible Examples <https://github.com/ansible/ansible-examples>`_
- `Ansible Baseline <https://github.com/ansible/ansible-baseline>`_

Reply:
@imjoseangel Done 👍